samples/template-operator/charts/template-operator/templates/rbac.authorization.k8s.io_v1_clusterrole_template-operator-manager-role.yaml

```diff
@@ -4,6 +4,39 @@ metadata:
   creationTimestamp: null
   name: template-operator-manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - create
+  - delete
```

Comment:
We have a software agent with elevated permissions running in the SKR cluster. Are we allowed for such a setup in the actual environment? Or it's just for the purpose of the demo/guide?

Reply:
this is a workaround introduced by @adityabhatia. This is not supposed to be permanent and we have to remove it soon.